Src/IronPython/Runtime/DictionaryStorage.cs

```diff
@@ -60,6 +79,17 @@ internal abstract class DictionaryStorage  {
 
         public abstract int Count { get; }
 
+        public bool IsMutable {
```

Comment:
Yes, it was meant for debugging purposes, and is a property so that it can be displayed in a _Watch_ window or inspected in an _Immediate_ window. `ConditionalAttribute` cannot be applied to properties; I would have to change it to a method. Since such method has to return `void` it would be something like
```C#
public void AssertMutable();
```
Is there a problem with leaving it as is?

Reply:
It's fine as it is. I guess the concern was someone trying to use this as a conditional to avoid a throw and then getting poor performance because it's throwing anyway. We could use `#if DEBUG`, but no big deal, it's all internal.